src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService service = new UserService();
```

Comment:
Not fixed prev comment.
Should be class level field. 
```suggestion
        UserService service = new UserService();
```